src/main/java/com/znsio/e2e/tools/Visual.java

```diff
@@ -112,22 +114,38 @@ private com.applitools.eyes.appium.Eyes instantiateAppiumEyes(String driverType,
         appEyes.addProperty(RUN_IN_CI, String.valueOf(getValueFromConfig(RUN_IN_CI)));
         appEyes.addProperty(TARGET_ENVIRONMENT, String.valueOf(getValueFromConfig(TARGET_ENVIRONMENT)));
         appEyes.addProperty("USER_NAME", USER_NAME);
+        String serverUrl = "curl -I --location --request GET '" + getValueFromConfig(APPLITOOLS.SERVER_URL, DEFAULT_APPLITOOLS_SERVER_URL)
+                + "api/sessions/renderinfo?apiKey=" + getApplitoolsAPIKey(isVisualTestingEnabled) + "'";
         try {
+            CheckApplitoolConnectivity(serverUrl);
             appEyes.open(innerDriver, appName + "-" + platform, testName);
-            LOGGER.info("instantiateAppiumEyes: Is Applitools Visual Testing enabled? - " + !appEyes.getIsDisabled());
+            LOGGER.info("" + "instantiateAppiumEyes: Is Applitools Visual Testing enabled? - " + !appEyes.getIsDisabled());
         } catch(IllegalArgumentException e) {
-            String message = String.format("Exception in instantiating Applitools for Web: '%s', Closing Web-driver instance", e.getMessage());
+            String message = String.format("Exception in instantiating Applitools for App: '%s', Closing driver instance", e.getMessage());
             LOGGER.error(message);
             innerDriver.quit();
             throw new VisualTestSetupException(message, e);
         }
+
         return appEyes;
     }
 
     private String getApplitoolsAPIKey(boolean isVisualTestingEnabled) {
         return isVisualTestingEnabled? getValueFromConfig(APPLITOOLS.API_KEY, null) : getValueFromConfig(APPLITOOLS.API_KEY, NOT_SET);
     }
 
+    private void CheckApplitoolConnectivity(String url){
```

Comment:
method name should be `checkApplitoolsConnectivity`

Reply:
Done